### ISSUE TITLE
Page not changing when navigating

### ISSUE THREAD

**Issue by sedubois** [relax/relax#348]
Similar to previously described, I create a minimal setup with pages "Home", "Discover", "About", all using a default template showing a header with navigation, which contains links to these 3 pages. I can open the site properly at `http://localhost:8080` and see the home page, but when e.g clicking "About", the URL changes but the content doesn't. If I reload the page, then the content changes properly.

**Comment by bruno12mota** [comment]
Yup that might be happening, still haven't double checked the entire frontend part
